Bluetooth: Samples: Add Advertising Coding Selection to Coded HR samples (#20753)
nrfconnect/sdk-nrf: samples/bluetooth/peripheral_hr_coded/README.rst

The sample demonstrates a basic Bluetooth® Low Energy Peripheral role functionality that exposes the Heart Rate GATT Service with LE Coded PHY support, which is not available in Zephyr Bluetooth LE Controller (See :ref:`ug_ble_controller` for more information).
Once it connects to a Central device, it generates dummy heart rate values.
You can use it together with the :ref:`bluetooth_central_hr_coded` sample.
The sample enables the :kconfig:option:`CONFIG_BT_EXT_ADV_CODING_SELECTION` Kconfig option to use the Advertising Coding Selection feature to explicitly set the required coding scheme for LE Coded PHY.

> The advertiser is configured to use the S=8 coding scheme, but this can be changed or removed depending on the applications needs through the advertising options.

Reviewer: ```suggestion
The advertiser is configured to use the S=8 coding scheme, but you can change or remove it through the advertising options, depending on the application's needs .
```

#. Wait until the Coded advertiser is detected by the Central.
   In the terminal window, check for information similar to the following::

>       Connected: xx.xx.xx.xx.xx.xx (random), tx_phy LE Coded, rx_phy LE Coded

Reviewer 2: Would be nice if you could update the docs to also mention the log line "Filters matched..."
